tests/unit/components/Asset.spec.js

```diff
@@ -102,9 +103,17 @@ describe('Asset', () => {
   });
 
   it('renders a `ReplayableVideoAsset` without poster variants', () => {
-    const videoAsset = mountAsset('video', { video }).find(ReplayableVideoAsset);
+    const identifier = 'video';
+    const videoAsset = mountAsset(identifier, { video }).find(ReplayableVideoAsset);
     expect(videoAsset.props('variants')).toBe(video.variants);
     expect(videoAsset.props('posterVariants')).toEqual([]);
+    expect(videoAsset.props('id')).toBe(identifier);
+    expect(videoAsset.props('alt')).toBe(video.alt);
+  });
+
+  it('renders a `ReplayableVideoAsset` without id if video does not provides an alt text', () => {
+    const videoAsset = mountAsset('video', { video: { ...video, alt: null } }).find(ReplayableVideoAsset);
+    expect(videoAsset.props('id')).not.toBe(video.id);
```

Comment:
This test technically passes, but it's not doing what is described in the test (and I wouldn't expect it to). You made a change so that id is always required, and you are passing one, so we shouldn't be testing that there isn't one in this scenario—and it only doesn't have one because the `video` object here doesn't have one, so this test is saying the real "video" idea is not equal to `video.id` (`expect("video").not.toBe(undefined)`)

I don't think there's anything wrong with the code at this point to be clear, just this test.

Reply:
I modified this component so many times that I forgot to delete this old test 🤦‍♀️  Thanks!